pkg/api/scylla/validation/scyllaoperatorconfig_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2023 ScyllaDB.
+
+package validation_test
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	scyllav1alpha1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1alpha1"
+	"github.com/scylladb/scylla-operator/pkg/api/scylla/validation"
+	"github.com/scylladb/scylla-operator/pkg/pointer"
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+func TestValidateScyllaOperatorConfig(t *testing.T) {
+	t.Parallel()
+
+	tt := []struct {
```

Comment:
it could also use a set of test cases for UnsupportedBashToolsImageOverride

Reply:
added